packages/frontend/src/hooks/useActivities.ts

```diff
@@ -4,11 +4,16 @@ import { METROM_API_CLIENT } from "../commons";
 import { useAccount, useChainId } from "wagmi";
 import { useQuery } from "@tanstack/react-query";
 import type { Address } from "viem";
+import type { HookBaseParams } from "../types/hooks";
+
+interface UseActivitiesParams extends HookBaseParams {}
+
+type QueryKey = [string, Address | undefined];
 
 const TIME_RANGE = 60 * 60 * 24 * 7; // 1 week
 
 // TODO: dynamic from and to
-export function useActivities(): {
+export function useActivities({ enabled = true }: UseActivitiesParams = {}): {
```

Comment:
Should we default this one to false to avoid silent calls to our backend?

Reply:
Not sure I get this, the `useActivities` gets used only when the account menu gets opened.

Reply:
Yeah but I don't understand the logic behind having an enabled flag set to `true` by default... boolean's default values are usually `false` for a reason.